test/InterchainTokenService.js

```diff
@@ -1508,8 +1584,8 @@ describe('Interchain Token Service', () => {
             await txPaused.wait();
         });
 
-        for (const type of ['lockUnlock', 'mintBurn', 'lockUnlockFee']) {
-            it(`Should be able to initiate an interchain token transfer via the interchainTransfer function on the service [${type}]`, async () => {
+        for (const type of ['lockUnlock', 'mintBurn', 'lockUnlockFee', 'mintBurnFrom']) {
+            it(`Should be able to initiate an interchain token transfer via the interchainTransfer function on the service with standard contract call [${type}]`, async () => {
                 const [token, tokenManager, tokenId] = await deployFunctions[type](`Test Token ${type}`, 'TT', 12, amount);
                 const sendAmount = type === 'lockUnlockFee' ? amount - 10 : amount;
                 const metadata = '0x00000000';
```

Comment:
4-byte of metadata or less does a plain transfer now. Need to add more data. For the receive transfer test, you can check the behaviour difference between empty/non empty data if it's not covered